pages/docs/project-management/auto-archive.mdx

```diff
@@ -0,0 +1,32 @@
+---
+description: Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
+---
+
+import { Callout } from "nextra-theme-docs";
+
+# Auto archive
+
+Auto archive allow you to automatically move to trash resources that have not been viewed based on your settings.
```

Comment:
"Auto archive allow you to automatically move resources to the trash that have not been recently viewed."

Reply:
👍